scio-core/src/main/scala/com/spotify/scio/ScioContext.scala

```diff
@@ -138,6 +149,39 @@ class ScioContext private[scio] (val options: DataflowPipelineOptions, testId: O
     new SCollectionImpl[T](p, this)
 
   // =======================================================================
+  // Extra artifacts - jars/files etc
+  // =======================================================================
+
+  /** Borrowed from Dataflow */
+  private def detectClassPathResourcesToStage(classLoader: ClassLoader) = {
+    require(classLoader.isInstanceOf[URLClassLoader], "Current ClassLoader is '" + classLoader + "' only URLClassLoaders are supported")
+    classLoader.asInstanceOf[URLClassLoader].getURLs.map{ url =>
+      Try {
+        new File(url.toURI).getAbsolutePath
+      }.getOrElse(throw new IllegalArgumentException("Unable to convert url '" + url + "' to file"))
+    }
+  }
+
+  /** Compute list of files to stage in dataflow */
+  private def getFilesToStage(extraLocalArtifacts: List[String]): List[String] = {
+    val finalLocalArtifacts = detectClassPathResourcesToStage(classOf[DataflowPipelineRunner].getClassLoader) ++ extraLocalArtifacts
+    //TODO: add logging, stats
+    finalLocalArtifacts.toList
+  }
+
+  /** *
```

Comment:
In IntelliJ pref, Editor -> Code Style -> Scala -> ScalaDoc, untick Use ScalaDoc indent for leading asterisk.